package.json

```diff
@@ -45,6 +46,7 @@
     ]
   },
   "devDependencies": {
+    "@types/validator": "^13.1.3",
```

Comment:
Preparation for URL validation update, it has however no function at all at the moment.